CONTRIBUTING.md

```diff
@@ -234,6 +234,13 @@ We believe that this makes it easier to see where a particular import comes from
 We prefer to import core traits, e.g. using `TryFrom` instead of
 `core::convert::TryFrom`.
 
+### Misc
+
+#### Avoid type aliases
+
+We prefer to avoid type aliases as they can obfuscate the underlying type and
+don't often provide significant additional type safety.
```

Comment:
"often" and "significant" feel like "weasel words" to me here: IMO they just don't. I would remove "can" for similar reasons. Maybe add a note about using a newtype wrapper as an alternative where an abstraction boundary is warranted/worth the complexity.

Reply:
Struck the weasel words and made a note about the newtype idiom.